src/main/java/com/twilio/Twilio.java

```diff
@@ -71,6 +73,35 @@ public static synchronized void init(final String username, final String passwor
         Twilio.setAccountSid(accountSid);
     }
 
+    /**
+     * Initialize the Twilio environment.
+     *
+     * @param username account to use
+     * @param password auth token for the account
+     * @param userAgentExtensions offers ability to extend the User-Agent string
+     */
+    public static synchronized void init(final String username, final String password, List<String> userAgentExtensions) {
+        Twilio.setUsername(username);
+        Twilio.setPassword(password);
+        Twilio.setUserAgentExtensions(userAgentExtensions);
+    }
+
+    /**
+     * Initialize the Twilio environment.
+     *
+     * @param username   account to use
+     * @param password   auth token for the account
+     * @param accountSid account sid to use
+     * @param userAgentExtensions offers ability to extend the User-Agent string
+     */
+    public static synchronized void init(final String username, final String password, final String accountSid,
+                                         List<String> userAgentExtensions) {
+        Twilio.setUsername(username);
+        Twilio.setPassword(password);
+        Twilio.setAccountSid(accountSid);
+        Twilio.setUserAgentExtensions(userAgentExtensions);
+    }
+
```

Comment:
Prefer to not overload `init` any more. Customers can simply call `setUserAgentExtensions` instead.

Reply:
addressed